statistician/env.yaml

```diff
@@ -0,0 +1,137 @@
+name: env
+channels:
+  - conda-forge
+dependencies:
+  - python=3.8.10
+  - libgdal=3.5.1
+  - gdal=3.5.1
+  - proj=9.0.1
+  - rasterio=1.3.2
```

Comment:
I like to try to comment exact version pins where possible, so future developers know what that they need to be aware of when updating.

Reply:
The idea here is to build a conda env with the minimum requirements so far we know that it'll be a) working b) without conflict, which is one major benefit of conda, it sorts out the dependencies when it can, and complains when it can't c) reproduce the results of current env we have been using when producing the current summary products, which is guaranteed by the "golden files" stored in aws. 

If we need, we can preserve the env locking list of very specific version infos, including path and sha, after the successful build. Then the same env can be built with the list. I'm not so sure if preserving the list is needed or not. IMHO, the version can be very loose as long as the results/data are the same. It's only for `odc-stats` case though, since any change made in `odc-stats` will be required to pull the latest env and pass the test, the same for the env build. Personally I don't prefer to staying unchanged, but being alarmed when the change introduce the problem and resolve it relatively easily.